### PR TITLE
Add now() to evaluation

### PR DESCRIPTION
**Pull request by yossioo**
Add option to fill stamp fields with `now()` method

Possible solution for #21 

Usage - two ways:
- put `now()` in `builtin_interfaces/Time` type field, e.g. `stamp` in `header`
- put `now().sec` and `now().nanosec` in relevant fields

rqt_now

### CODE REVIEW

**Comment by yossioo** [comment]
@dirk-thomas Any idea if someone is available to examine the request?

**Comment by dirk-thomas** [comment]
@yossioo I am not the maintainer of this repository anymore. @mjcarroll should review this ticket.

**Comment by yossioo** [comment]
Thanks!

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**
